src/routes/styleguide/+page.svelte

```diff
@@ -18,7 +25,7 @@
 		<li><button class="link">link button</button></li>
 	</ul>
 </section>
-<section>
+<!-- <section id="search_button">
 	<h2>Search Buttons</h2>
```

Comment:
Why is this commented out?

Reply:
so i didnt need to zoom all the way out to test card component

Reply:
shouldn't be committing it with the comments on.